docs-src/cli/server/start-dev.md

```diff
@@ -7,9 +7,6 @@ tags:
 	- cli
 ---
 
-The `temporal server start-dev` command starts the Temporal Server on `localhost:7233`.
```

Comment:
uh oh why is this removed?

Reply:
It shouldn't have been. Let me go add that back.

Reply:
So it's showing up in my branch on VSCode but not on GitHub?